plugins/upload-assets-to-s3/test/tasks/upload-assets-to-s3.test.ts

```diff
@@ -51,10 +52,10 @@ describe('upload-assets-to-s3', () => {
     process.env.NODE_ENV = 'production'
 
     await task.run()
-
-    const s3 = mocked(mockedAWS.S3.mock.instances[0])
-    expect(s3.upload).toHaveBeenCalledTimes(2)
-    expect(s3.upload.mock.calls[0][0]).toHaveProperty('Key', 'testdir/nested/test.js.gz')
+    const s3 = mockedS3Client.mock.instances[0]
+    const putObjectCommand = mockedPutObjectCommand.mock.instances[0]
+    expect(s3.send).toHaveBeenCalledTimes(1)
+    // expect(putObjectCommand.mock.calls[0][0]).toHaveProperty('Key', 'testdir/nested/test.js.gz')
```

Comment:
@ivomurrell did you get to the bottom of how to test this?

Reply:
I forgot 🙊 I'll do that today